README.md

```diff
@@ -161,6 +161,11 @@ SGX_MODE=SW make
 
 ## Launch
 
+Before executing `launch.py`, the following Python libraries need to be installed
+```
+pip install python-dotenv pycurl docker toml
+```
+
```

Comment:
Thanks!

Shall we have a `requirements.txt` for installation?

Reply:
I think maybe not necessary. Not too many packages here.